pinot-core/src/main/java/org/apache/pinot/core/query/aggregation/function/PercentileRawTDigestAggregationFunction.java

```diff
@@ -59,11 +64,12 @@ public AggregationFunctionType getType() {
   @Override
   public String getResultColumnName() {
     final double percentile = _percentileTDigestAggregationFunction._percentile;
+    final double compressionFactor = _percentileTDigestAggregationFunction._compressionFactor;
     final int version = _percentileTDigestAggregationFunction._version;
     final String type = getType().getName().toLowerCase();
 
     return version == 0 ? type + (int) percentile + "(" + _expression + ")"
-        : type + "(" + _expression + ", " + percentile + ")";
+        : type + "(" + _expression + ", " + percentile + ", " + compressionFactor + ")";
```

Comment:
Would this change break some existing readers depending on this column name?

Reply:
Oh i see, do you recommend not adding the compressionFactor at all to this columnName then?

Reply:
+1 I just gave a similar comment too. Result column name should not change imo.

Reply:
done